src/simularium/VisData.ts

```diff
@@ -164,6 +166,147 @@ class VisData {
         };
     }
 
+    public static parseBinary(data: ArrayBuffer): ParsedBundle {
+        const parsedAgentDataArray: AgentData[][] = [];
+        const frameDataArray: FrameData[] = [];
+
+        const enc = new TextEncoder(); // always utf-8
+        const eofPhrase = enc.encode("\\EOFTHEFRAMEENDSHERE");
```

Comment:
Maybe worth saving this string `"\\EOFTHEFRAMEENDSHERE"` as a variable since it's used here and in `parseBinaryNetData`